docs/releasenotes/release-1.136.md

```diff
@@ -0,0 +1,44 @@
+
+** This version is not yet released; this document is gathering release notes for the future release **
+
+*   Special shout-outs to @acpana, @anhdle-sso, @cheftako, @gemmahou, @jingyih, @justinsb, @xiaoweim, @yuwenma for their contributions to this release.
+
+## New Beta Resources
+
+*   `AssetFeed`
+*   `BigQueryReservationAssignment`
+*   `CloudDeployDeliveryPipeline`
+*   `ComposerEnvironment`
+*   `NetworkSecurityAuthorizationPolicy`
+*   `NetworkSecurityClientTlsPolicy`
+*   `NetworkSecurityServerTlsPolicy`
+*   `RunJob`
```

Comment:
This is not a new resource, but a beta migration (and has not been finished yet).